### PR TITLE
fix: Update readme examples

### PR DESCRIPTION
**Pull request by tmathern**
Follow-up of https://github.com/contentauth/c2pa-node/pull/86

Update readme examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,11 +289,15 @@ To avoid loading the entire asset into memory (or for file types other than JPEG
...
 
+// get the asset full path
+const asset = {
+  path: resolve('to-be-signed.jpg'),
+};
+// define a location where to place the signed asset
+const outputPath = resolve('signed.jpg');
+
+// create a signing function
 async function sign(asset, manifest) {
```
Comment (tmathern) [inline README.md:300]: asset/output path moved out since they'll be passed in as params

File: README.md [hunk 2]
```diff
@@ -231,9 +231,41 @@ If you have an asset file's data loaded into memory, you can sign the the asset
...
+      },
+    ],
+    },
+  { vendor: 'cai' },
+);
+
+// create a signing function
 async function sign(asset, manifest) {
-  const buffer = await readFile('to-be-signed.jpg');
-  const asset: Asset = { buffer, mimeType: 'image/jpeg' };
```
Comment (tmathern) [inline README.md:236]: asset/buffer moved out since they'll be passed in as params to the function

File: README.md [hunk 3]
```diff
@@ -303,9 +371,13 @@ async function createLocalSigner() {
   };
 }
 
+// read the asset
+const buffer = await readFile('to-be-signed.jpg');
+// asset mimetype must match the asset type ebing read
+const asset: Asset = { buffer, mimeType: 'image/jpeg' };
+
+// create a signing function
 async function sign(asset, manifest) {
```
Comment (tmathern) [inline README.md:380]: asset/buffer moved out since they'll be passed in as params to the sign function

File: README.md [hunk 4]
```diff
@@ -231,9 +231,41 @@ If you have an asset file's data loaded into memory, you can sign the the asset
...
+          version: '1.0.0',
+        },
+      },
+    ],
+    },
+  { vendor: 'cai' },
+);
+
+// create a signing function
 async function sign(asset, manifest) {
```
Comment (tmathern) [inline README.md:268]: asset/buffer moved out since they'll be passed in as params to the sign function

File: README.md [hunk 5]
```diff
@@ -275,7 +312,37 @@ async function sign(asset, manifest) {
   });
 }
 
-sign(asset, manifest);
+// build a manifest to use for signing
+const manifest = new ManifestBuilder(
```
Comment (tmathern) [inline README.md:316]: A bit repetitive, but it makes the examples copy-pastable to have the manifest buildet here too

File: README.md [hunk 6]
```diff
@@ -317,7 +389,37 @@ async function sign(asset, manifest) {
...
```
Comment (tmathern) [inline README.md:393]: A bit repetitive, but it makes the examples copy-pastable to have the manifest buildet here too

File: README.md [hunk 7]
```diff
@@ -36,8 +36,8 @@ Contents:
 ### Prerequisites
 
 You must install:
-- A [supported version of Node](https://github.com/neon-bindings/neon#platform-support).
-- [Rust](https://www.rust-lang.org/tools/install) 
+- A [supported version of Node](https://github.com/neon-bindings/neon#platform-support);
```
Comment (crandmck) [inline README.md:39]: This should end in a period, per Adobe Style Guide https://writing.adobe.com/58b352410/p/646bcb-common-questions/t/186612

**Review by crandmck — APPROVED** [review]
One small punctuation correction, otherwise LGTM.

File: README.md (hunk 7 above)
Comment (tmathern) [reply README.md]: Fixed!

**Pull request merged**